submission_rules.adoc

```diff
@@ -269,8 +269,7 @@ MLCommons provides a web UI [TODO: URL] for verifying scores contained in the ta
 #### Post-submission grace period (submission deadline + 60 minutes):
 MLPerf will allow submissions for up to 60 minutes after the published deadline without explanation or penalty. The end of this grace period constitutes a hard deadline which will be strictly enforced.
 
-If a submitter cannot upload a valid submission tarball by the hard deadline due to some defect in the submission checker, a hash of the tarball must be sent to the submitters mailing list before the submission deadline and the submission tarball must be mailed to the submitters mailing list not later than 30 minutes from the submission deadline. An issue must also be raised on the relevant GitHub repository detailing the failure of the submission checker before the submission deadline. The burden is on the submitter to demonstrate that the submission could not be uploaded to via the web GUI because the checker was defective, and that the tarball passes the submission checker once the defect has been addressed. No other changes to the tarball other than those directly related to the submission checker fix will be allowed.
-
+If a submitter cannot upload a valid submission tarball by the hard deadline due to some defect in the submission checker, a hash of the tarball must be sent to the submitters mailing list before the submission deadline and the submission tarball must be mailed to the submitters mailing list not later than 30 minutes from the submission deadline. An issue must also be raised on the relevant GitHub repository detailing the failure of the submission checker before the submission deadline. The burden is on the submitter to demonstrate that the submission could not be uploaded to via the web GUI because the checker was defective, and that the tarball passes the submission checker once the defect has been addressed. No other changes to the tarball other than those directly related to the submission checker fix will be allowed. For example, if the submission checker did not mandate one or more of the required compliance tests or any other mandatory file due to the bug, they can be later added to the submission tarball. But if the submission checker has already thrown out an error for one or missing compliance tests or any other mandatory file even before the bug is fixed, then they cannot be after wards added to the submission tarball.
```

Comment:
```suggestion
If a submitter cannot upload a valid submission tarball by the hard deadline due to some defect in the submission checker, a hash of the tarball must be sent to the submitters mailing list before the submission deadline and the submission tarball must be mailed to the submitters mailing list not later than 30 minutes from the submission deadline. An issue must also be raised on the relevant GitHub repository detailing the failure of the submission checker before the submission deadline. The burden is on the submitter to demonstrate that the submission could not be uploaded to via the web GUI because the checker was defective, and that the tarball passes the submission checker once the defect has been addressed. No other changes to the tarball other than those directly related to the submission checker fix will be allowed. For example, if the submission checker did not mandate one or more of the required compliance tests or any other mandatory file due to the bug, they can be later added to the submission tarball. But if the submission checker has already reported an error for one or missing compliance tests or any other mandatory file even before the bug is fixed, then they cannot be afterwards added to the submission tarball.
```

Reply:
(changed "thrown out an error" to "reported an error" and "after wards" to "afterwards")

Reply:
Thank you @DilipSequeira for the changes.